packages/core/src/oot/actors/Player.S

```diff
@@ -195,3 +195,15 @@ PATCH_START 0x8007b8b8
    nop
   nop
 PATCH_END
+
+/*
```

Comment:
No replace comment

Reply:
I don't understand. There is a replace comment. Do you want me to remove it? Why?

Reply:
Yeah it's noise, remove it